core/db/migrate/20170105164636_add_partial_index_apply_automatically_promotions.rb

```diff
@@ -0,0 +1,15 @@
+class AddPartialIndexApplyAutomaticallyPromotions < ActiveRecord::Migration[5.0]
+  def change
+    condition = if connection.adapter_name =~ /sqlite/i
+                  "apply_automatically = 1"
```

Comment:
I believe SQLite actually uses `'t'` as its true value.

It's probably safer and simpler here to use

``` ruby
condition = "apply_automatically = #{connection.quoted_true}"
```

Otherwise :+1: 

It's worth noting that MySQL and its forks don't support partial indexes, but there should be no harm (and still considerable gain) in it adding this index without the condition.